includes/snmp.inc.php

```diff
@@ -264,6 +282,29 @@ function snmp_get($device, $oid, $options = null, $mib = null, $mibdir = null)
 }//end snmp_get()
 
 
+function snmp_get_next($device, $oid, $options = null, $mib = null, $mibdir = null)
```

Comment:
I'm not sure I see the benefit of this? 
I still need to count the number of radio interfaces to make the information available for graphing, I will also need to sort the result of each snmp_walk into the appropriate array for mib polling and rrd ds insertion.

I will have to do lots of hoop-jumping and write at least a few very device specific functions in order to avoid the creation and use of one very generic function which exposes a basic NET-SNMP (snmpgetnext) function to LibreNMS.